src/foam/u2/CheckBox.js

```diff
@@ -66,11 +66,11 @@ foam.CLASS({
         margin: 8px;
         padding: 8px;
       }
-
+      
       ^label {
         color: #444;
         flex-grow: 1;
-        margin-right: 12px;
+        margin-left: 12px;
```

Comment:
It should've been `margin-left` from the start, adding a margin to the right doesn't do anything as the checkbox is to the left. (I added it in last month so probably not too many users of this just yet).